pages/france.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Monstera Deliciosas</title>
+    <link rel="stylesheet" href="/css/france.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="imgLogo">
+                <img src="/images/helloWorld.png" alt="Hello World!">
+                
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="../index.html">Jamaica</a></li>
+                    <li><a href="france.html">France</a></li>
+                    <li><a href="uk.html">United Kingdom</a></li>
+                    <li><a href="newzealand.html">New Zealand</a></li>
+                  </ul>
+            </nav>
+            <h3>“A walk about Paris will provide lessons in history, beauty, and in the point of Life.” – Thomas Jefferson</h3>
+        </header>
+        <div id="countryImages">
+            <img class="countryImages" src="/images/eiffel_tower.jpg" alt="Eiffel Tower">
+            <img class="countryImages" src="/images/ArcTriomphe.jpg" alt="Arc de Triomphe">
+        
+            <img class="countryImages" src="/images/versailles.jpg" alt="Versailles">
+            <img class="countryImages" src="/images/louvre.jpg" alt="Louvre">
+        </div>
+        <div>
+            <h1 style "color: red;">France</h1>
```

Comment:
need to remove the style "color: red;"

This is causing an issue with the HTML validator.